examples/Basic.hs

```diff
@@ -22,18 +22,26 @@ mean xs = sum' xs / fromIntegral (length xs)
 
 main :: IO ()
 main = do
-    handle <- forkServer "localhost" 8000
-    counter <- getCounter "iterations" handle
-    label <- getLabel "args" handle
-    event <- getDistribution "runtime" handle
+    handle <- forkServer "0.0.0.0" 8000
+    baseCounter  <- getCounter "iterations" [] handle
```

Comment:
I don't think it makes sense for a counter to be accessed with only some dimensions. What does it mean to increment a counter that has been given e.g. a URL and a HTTP status code dimension without providing values for those when incrementing it?